doc/source/changes/version_0_22.rst.inc

```diff
@@ -208,3 +218,16 @@ Fixes
 
 * fixed array creation functions not raising an exception when called with wrong syntax func(axis1, axis2, ...)
   instead of func([axis1, axis2, ...]) (closes :issue:`203`).
+
+* fixed adding new sheets in a workbook: a new sheet is appended instead of prepended (closes :issue:`229`).
```

Comment:
this is a tad misleading: if you read up to the ":", you think it did not work at all. I would phrase this like: "fixed position of added sheets in excel workbook: new sheets are appended instead of prepended."